cypress/e2e/tests/pages/explorer/dashboard/events.spec.ts

```diff
@@ -0,0 +1,200 @@
+import ClusterDashboardPagePo from '@/cypress/e2e/po/pages/explorer/cluster-dashboard.po';
+import { EventsPagePo } from '@/cypress/e2e/po/pages/explorer/events.po';
+import { generateEventsDataSmall } from '@/cypress/e2e/blueprints/explorer/cluster/events';
+import HomePagePo from '@/cypress/e2e/po/pages/home.po';
+import LoadingPo from '~/cypress/e2e/po/components/loading.po';
+
+const clusterDashboard = new ClusterDashboardPagePo('local');
+const events = new EventsPagePo('local');
+const uniquePod = 'aaa-e2e-test-pod-name';
+const podNamesList = [];
+
+describe('Events', { testIsolation: 'off', tags: ['@explorer', '@adminUser'] }, () => {
+  before(() => {
+    cy.login();
+  });
+
+  beforeEach(() => {
+    cy.createE2EResourceName('namespace1').as('namespaceName1');
+    cy.createE2EResourceName('namespace2').as('namespaceName2');
+  });
+
+  describe('List', { tags: ['@vai'] }, () => {
+    before('set up', () => {
+      cy.updateNamespaceFilter('local', 'none', '{\"local\":[]}');
+
+      cy.createE2EResourceName('namespace1').then((nsName1) => {
+        // create namespace
+        cy.createNamespace(nsName1);
+
+        // create pods
+        let i = 0;
+
+        while (i < 125) {
+          const podName = `e2e-${ Cypress._.uniqueId(Date.now().toString()) }`;
+
+          cy.createPod(nsName1, podName, 'nginx:latest', false).then(() => {
```

Comment:
is this done to create the large number of events?

Reply:
correct